io.github.leeroymanea.synology-note-station.yaml

```
@@ -0,0 +1,42 @@
+# copied/inspired from https://github.com/flathub/com.synology.SynologyDrive/blob/master/com.synology.SynologyDrive.yaml
+id: io.github.leeroymanea.synology-note-station
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: launch.sh
+finish-args:
+  # needed for launch.sh (code from synology)
+  - --filesystem=home
+  - --share=network
+  - --socket=x11
+  - --share=ipc
+modules:
+  - name: synology-note-station
+    buildsystem: simple
+    sources:
+      - type: file
+        sha256: a4c763bddda21bd211b0c8afdcff7e72a3edddc2392470fec8ce8123a0531751
+        url: "https://global.synologydownload.com/download/Utility/NoteStationClient/2.2.4-703/Ubuntu/x86_64/synology-note-station-client-2.2.4-703-linux-x64.deb"        
+      - type: file
+        path: io.github.leeroymanea.synology-note-station.metainfo.xml
+    build-commands:
+       # unzip folder
+      - ar x synology-note-station-client-2.2.4-703-linux-x64.deb
+      - tar -xf data.tar.xz
+      # move to app location
+      - mv usr/share /app/share
+      - mv opt /app/opt
+      # rename some files
+      - mv /app/share/applications/synology-note-station-client.desktop /app/share/applications/io.github.leeroymanea.synology-note-station.desktop
+      - find /app/share/icons/hicolor -type f -execdir sh -c 'mv "$0" io.github.leeroymanea.synology-note-station.png' {} \;
+      # link launch.sh
+      - mkdir -p /app/bin/
+      - ln -s /app/opt/synology-note-station/launch.sh /app/bin/launch.sh
+      # update desktop file
+      # fix issue => error: value "2.0" for key "Version" in group "Desktop Entry" is not a known version
+      - desktop-file-edit --remove-key=Version /app/share/applications/io.github.leeroymanea.synology-note-station.desktop
+      - desktop-file-edit --set-key=Exec --set-value=launch.sh /app/share/applications/io.github.leeroymanea.synology-note-station.desktop
+      - desktop-file-edit --set-key=Icon --set-value=io.github.leeroymanea.synology-note-station.png /app/share/applications/io.github.leeroymanea.synology-note-station.desktop
```

```suggestion
      - desktop-file-edit --set-icon=io.github.leeroymanea.synology-note-station /app/share/applications/io.github.leeroymanea.synology-note-station.desktop
```

1. there is `--set-icon`
2. it NEVER has the extension. it is the icon **name**

---

Thank you for your advice. I will change it.

Where do i find the resources to that? 

I struggled a lot to bring everything to work with chatgpt.